packages/vite/src/node/server/http.ts

```diff
@@ -2,20 +2,18 @@ import fs from 'fs'
 import path from 'path'
 import { Server as HttpServer } from 'http'
 import { ServerOptions as HttpsServerOptions } from 'https'
-import { ServerOptions } from '..'
+import { ResolvedConfig, ServerOptions } from '..'
 import { Connect } from 'types/connect'
 
 export async function resolveHttpServer(
   { https = false, proxy }: ServerOptions,
```

Comment:
**suggestion (non-blocking):** Looks like this `https` property is no in use anymore, therefor you should remove it